CODEOWNERS

```
@@ -0,0 +1,5 @@
+* @marcoceppi
+* @castrojo
+/docs/ @bigpod98
+/docs/ @bsherman
+/docs/ @EinoHR
```

```suggestion
/docs/ @bigpod98 @bsherman @EinoHR @marcoceppi @castrojo 
* @marcoceppi @castrojo
```

CODEOWNERS is processed from top to bottom for first match. The previous would only ever match you and I